pkgs/applications/window-managers/sway/osd.nix

```diff
@@ -21,26 +27,47 @@ rustPlatform.buildRustPackage {
     hash = "sha256-MJuTwEI599Y7q+0u0DMxRYaXsZfpksc2csgnK9Ghp/E=";
   };
 
-  cargoHash = "sha256-pExpzQwuHREhgkj+eZ8drBVsh/B3WiQBBh906O6ymFw=";
+  cargoDeps = rustPlatform.fetchCargoTarball {
+    inherit src;
+    name = "${pname}-${version}";
+    hash = "sha256-pExpzQwuHREhgkj+eZ8drBVsh/B3WiQBBh906O6ymFw=";
+  };
 
   nativeBuildInputs = [
     wrapGAppsHook
     pkg-config
+    meson
+    rustc
+    cargo
+    ninja
+    rustPlatform.cargoSetupHook
   ];
 
   buildInputs = [
+    coreutils
```

Comment:
```suggestion
```
that's unused and not required for postPatch to work

Reply:
corrected